test/utils.ts

```diff
@@ -1,6 +1,6 @@
 import { ethers } from "hardhat";
-import { BigNumber, Signer } from "ethers"
-import { ERC20 } from "../typechain"
+import { BigNumber, Signer } from "ethers";
```

Comment:
also not necessary for this time, but what linter are you using?

lets add another another test for linting in the future, so that these linting changes are performed in every PR
https://www.notion.so/54056ca820b44017b2ec73620324ebff?v=1e2ac750ac35462a8f524b62f8234ab2&p=9f02adb8769d4ed6b9bf80ceff3c4cf6